### DISCUSSION TITLE
Default data sets once installed

### DISCUSSION THREAD

**Discussion by emilycornes** [ckan/ckan#6209] [General]
Hi everyone,

I silly question, but my IT area has installed CKAN for me to begin using. However, there are a number of data sets already in this template. Is this right?

![image](https://user-images.githubusercontent.com/60957661/124050974-b7c53a80-da5a-11eb-9f86-ec61cf810137.png)

**Comment by Zharktas** [comment]
They probably ran cli command for creating test data https://docs.ckan.org/en/2.9/maintaining/cli.html#seed-create-test-data-in-the-database
